lib/utils/helpers.coffee

```diff
@@ -52,3 +55,24 @@ exports.sudo = (command, message) ->
 		console.log('Type your computer password to continue')
 
 	return president.executeAsync(command)
+
+# Loads '.resin-sync.yml' configuration from 'source' directory.
+# Returns the configuration object on success
+#
+# TODO: Use 'config.load()' method from `resin sync` when resin sync gets
+# integrated into resin CLI
+exports.loadConfig = (source) ->
+	configPath = path.join(source, '.resin-sync.yml')
```

Comment:
Good point @jviotti , moved `loadConfig()` as a private function in `sync.coffee.